src/style.css

```diff
@@ -1 +1,80 @@
-/* Styles go here */
+
+body {
+  --link-color: #00acdc;
+  --logo-height: 40px;
+  --logo-margin: 10px;
+  margin: 0;
+}
+
+header {
+  display: flex;
+  position: relative;
+  align-items: center;
+  justify-content: space-between;
+  flex-direction: row;
+  height: fit-content;
+}
+
+.nav__list {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  list-style-type: none;
+  margin-right: 50px;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.logo {
+  margin: 10px 0 10px 50px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 40px;
+  height: var(--logo-height);
+}
+
+.nav__item {
+  display: flex;
+  margin-left: 20px;
+  line-height: 14px;
+  text-transform: uppercase;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: #000;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  height: calc(var(--logo-height) + (var(--logo-margin)*2));
```

Comment:
use padding: 23px 0; instead of hardcoding height for elements with content inside